tests/NSubstitute.Analyzers.Tests.Shared/DiagnosticAnalyzers/DiagnosticVerifier.cs

```diff
@@ -173,22 +185,14 @@ protected Project CreateProject(string[] sources)
                     .CurrentSolution
                     .AddProject(projectId, TestProjectName, TestProjectName, Language)
                     .WithProjectCompilationOptions(projectId, compilationOptions)
-                    .AddMetadataReference(projectId, CorlibReference)
-                    .AddMetadataReference(projectId, SystemCoreReference)
-                    .AddMetadataReference(projectId, CodeAnalysisReference)
-                    .AddMetadataReference(projectId, NSubstituteReference)
-                    .AddMetadataReference(projectId, ValueTaskReference)
-                    .AddMetadataReference(projectId, systemRuntimeReference)
-                    .AddMetadataReference(projectId, systemThreadingTasksReference)
-                    .AddMetadataReferences(projectId, GetAdditionalMetadataReferences());
-
-                var count = 0;
-                foreach (var source in sources)
+                    .AddMetadataReferences(projectId, MetadataReferences.Concat(GetAdditionalMetadataReferences()));
```

Comment:
Tests run optimization. AppVeyor Unix machine is significantly slower than Windows one, so with this change we gain significant performance boost